src/core/server/config/env.ts

```diff
@@ -36,11 +37,9 @@ interface EnvironmentMode {
 }
 
 export interface EnvOptions {
-  config?: string;
-  kbnServer?: any;
-  packageInfo: PackageInfo;
-  mode: EnvironmentMode;
-  [key: string]: any;
+  configs: string[];
```

Comment:
__note:__ Kibana can rely on multiple configs at the same time (content will be merged).